applications_tests/gls_navier_stokes_2d/gls_static_bubble_surface_tension_force.output

```diff
@@ -0,0 +1,118 @@
+Running on 1 MPI rank(s)...
+   Number of active cells:       4096
+   Number of degrees of freedom: 12675
+   Volume of triangulation:      0.0064
+   Number of VOF degrees of freedom: 4225
+
+*****************************************************************
+Transient iteration : 1        Time : 0.001    Time step : 0.001    CFL : 0       
+*****************************************************************
+Newton iteration: 0  - Residual:  0
+  -Tolerance of iterative solver is : 1e-08
+  -Iterative solver took : 0 steps 
```

Comment:
Usually tests are "quiet", why do you need the verbose here?

Reply:
corrected.